pysondb/cli.py

```diff
@@ -135,18 +147,72 @@ def verify_file(file_data, refer_keys, filename):
         print("The number keys in DB entries does not match")
 
 
-def main():
-    fire.Fire(
-        {
-            "create": create_if_not_exist,
-            "display": display,
-            "delete": delete,
-            "convert": convert,
-            "converttocsv": convert_db_to_csv,
-            "merge": merge,
-        }
+def set_parser():
+    parser = argparse.ArgumentParser()
```

Comment:
The `ArgumentParser` should now accept `argv` as an argument
`argparse.ArgumentParser(argv)`